bin/scheduler.py

```diff
@@ -24,9 +24,8 @@
 
 for algorithm, algo_config in services_config.iteritems():
     scheduler_options = algo_config['scheduler_options']
-    plugin = getattr(poll_tasks, scheduler_options['plugin'])
     CELERYBEAT_SCHEDULE[algorithm] = {
         'task': 'lib.app.task_runner',
         'schedule': timedelta(seconds=scheduler_options['interval_secs']),
-        'args': (plugin, scheduler_options['plugin_args'])
+        'args': (PollTask, {'plugin_name': scheduler_options['plugin']})
```

Comment:
sorry, actually if we leave plugin we could have two times the same algorithm but with different schedules ... so let's leave it, but maybe we default to algorithm if plugin is not given ...